DTS-STN/Service-Canada-Labs
# removing links to projects that dont have project cards

# Description

Just removing the ability to navigate to projects that arent included in SC Labs any more

## Test Instructions

1. launch the app
2. navigate to http://localhost:3000/en/projects/life-journeys
3. navigate to http://localhost:3000/en/projects/digital-centre
4. navigate to http://localhost:3000/en/projects/virtual-assistant
5. verify the above links all lead to the 404 page

## Checklist


## Product and Sprint Backlog

[ADO](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities)
